packages/react-scripts/scripts/coalesceLocales.js

```diff
@@ -16,77 +16,107 @@ exports.coalesceLocales = paths => {
+          Object.keys(strings)
+        );
+        if (collisions && collisions.length > 0) {
           collisions.forEach(collisionKey => {
-            if (allLocales[locale][ns][collisionKey] !== strings[collisionKey]) {
-              collisionReport.collisions.push({key:collisionKey,locale,ns,oldValue: allLocales[locale][ns][collisionKey], newValue: strings[collisionKey]})
+            if (
+              allLocales[locale][ns][collisionKey] !== strings[collisionKey]
+            ) {
+              collisionReport.collisions.push({
+                key: collisionKey,
+                locale,
+                ns,
+                oldValue: allLocales[locale][ns][collisionKey],
+                newValue: strings[collisionKey],
+              });
               // console.error(`COLLISION: key=${collisionKey} locale=${locale} namespace=${ns} "${strings[collisionKey]}" overrides "${allLocales[locale][ns][collisionKey]}"`)
             } else {
-              collisionReport.potentialCollisions.push({key:collisionKey,locale,ns,oldValue: allLocales[locale][ns][collisionKey], newValue: strings[collisionKey]})
+              collisionReport.potentialCollisions.push({
+                key: collisionKey,
+                locale,
+                ns,
+                oldValue: allLocales[locale][ns][collisionKey],
+                newValue: strings[collisionKey],
+              });
             }
-          })
+          });
         }
         allLocales[locale][ns] = { ...allLocales[locale][ns], ...strings };
       });
     });
   });
   if (collisionReport.potentialCollisions.length > 0) {
     console.warn(`WARNING: There were ${collisionReport.potentialCollisions.length} potential collisions detected when coalescing locales, the values were the same, but could diverge in the future:
-      To see a list of all potential collisions, turn on debugging with "DEBUG=coalesceLocales" before the command you just ran `)
-    collisionReport.potentialCollisions.forEach(({ key, locale, ns, newValue }) => debug(`\tkey=${key} namespace=${ns} locale=${locale} value="${newValue}"`))
+      To see a list of all potential collisions, turn on debugging with "DEBUG=coalesceLocales" before the command you just ran `);
+    collisionReport.potentialCollisions.forEach(
+      ({ key, locale, ns, newValue }) =>
+        debug(
+          `\tkey=${key} namespace=${ns} locale=${locale} value="${newValue}"`
+        )
+    );
   }
   if (collisionReport.collisions.length > 0) {
-    console.error(`ERROR: The following ${collisionReport.collisions.length} collisions were detected when coalescing locales:`, collisionReport.collisions)
-    collisionReport.collisions.forEach(({ key, locale, ns, newValue, oldValue }) => console.error(`\tkey=${key} namespace=${ns} locale=${locale} newValue="${newValue}" oldValue="${oldValue}"`))
+    console.error(
+      `ERROR: The following ${collisionReport.collisions.length} collisions were detected when coalescing locales:`,
```

Comment:
do you have different line length overrides causing all this formatting churn? or do we?

Reply:
The create-react-app has a prettier config that does not have a line length, so it defaults to 80. The one used by everyone is this: https://vscode.dev/github/fs-webdev/eslint-config-frontier-react/blob/jordi/import/prettierConfig.js#L2 and it has a printWidth of 120.

Looks like Joey turned off lint-staged 4 years ago which did the prettier stuff on commit. So, I am just doing what the repository wants and everyone else is going against it. :P

Reply:
The less changes we can make to this repo, the better. It will make merging upstream changes from facebook easier in the future. 

Only God knows why I turned of lint-staged 4 years ago.

Doing what the repository wants is correct, we've all diverged because of our editor configurations.

Reply:
Joey, I think you turned off lint-staged to prevent so many people from doing no-verify pushes (but so many still do no-verify pushes all the time in zion just because)

Reply:
But this is in create-react-app. This was 4 years ago when we were still defining how we want to do things. Also, I'm willing to bet that 4 years ago there was a bunch of tests that ran on pre-commit hook, and I didn't want testing of cra's monorepo to happen when all I cared about is react-scripts.

Reply:
Ah, right.